pkg/applyconfiguration/execer.go

```diff
@@ -32,7 +33,11 @@ func ExecApplyConfiguration(ctx context.Context, binaryPath, inputDirectory, out
 		"--output-dir", outputDirectory,
 	}
 
-	cmd := exec.CommandContext(ctx, binaryPath, args...)
+	// TODO prove that the timeout works if the process captures sig-int
```

Comment:
maybe the applyconfig library could offer sigterm handling

Reply:
> maybe the applyconfig library could offer sigterm handling

Probably reasonable, but not here.